### PR TITLE
Slide style update

### PR DESCRIPTION
**Pull request by Nikomus**
Adds some new custom items to use, some variable tidying and spelling changes.

image

### CODE REVIEW

File: src/Exercises.elm [hunk 1]
```diff
@@ -465,7 +464,7 @@ slideContent section =
                     , bullet "If not, can you look up the constructs?"
                     ]
                 , slideP "Why are the syntactic constructs unfamiliar?"
-                , slideP "Are they ideosyncratic to this language or code base?"
+                , slideP "Are they idiosyncratic to this language or code base?"
```
Comment (katjam) [inline src/Exercises.elm:467]: Loving your spell checker. Than you!

File: src/Exercises.elm [hunk 2]
```diff
@@ -718,6 +717,11 @@ slideContent section =
 -- Markup helpers
 
 
+codeReadingWordmark : CustomContent
+codeReadingWordmark =
```
Comment (katjam) [inline src/Exercises.elm:721]: Is wordmark a thing?

Comment (katjam) [reply src/Exercises.elm:721]: I like it

**Review by katjam — APPROVED** [review]
Yes. I like these changes. Will merge. Thank you! :rocket:

File: src/Exercises.elm (hunk 1 above)
Comment (Nikomus) [reply src/Exercises.elm:467]: I can also sometimes spell... 😅

File: src/Exercises.elm (hunk 2 above)
Comment (Nikomus) [reply src/Exercises.elm:721]: Yep!

**Pull request merged**